packages/pwa-kit-cli/src/ssr/server/build-dev-server.js

```diff
@@ -173,6 +173,11 @@ export const DevServerMixin = {
         })
     },
 
+    // eslint-disable-next-line no-unused-vars
+    _onAllRequestsBeforeWebpackReady(req, res, next) {
```

Comment:
Looks good to me. Can we change the name to describe what this fn does though please? It doesn't do anything with "all requests"!

```suggestion
    _redirectToLoadingScreen(req, res, next) {
```